charts/jindofsx/templates/worker/statefulset.yaml

```diff
@@ -180,6 +180,9 @@ spec:
           {{- range .Values.ufsVolumes }}
```

Comment:
Could you update chart version and changelog?

Reply:
done